pvp_xray_exemptions.json

```diff
@@ -17,12 +17,6 @@
           "errors": [
             "Runtime/Materials/Overlay.mat"
           ]
-        },
-        "PVP-150-1": {
-          "errors": [
-            "UnityEngine.Formats.Alembic.Util.AlembicRecorder: Recording: unexpected <returns>; use <value> instead",
```

Comment:
I see you fixed one of them, is there more ?

Reply:
One is in AlembicExporter_impl.cs, the other is in AlembicStreamPlayer.cs.
I just removed one redundant angle bracket in the doc for `public bool ReloadStream(bool createMissingNodes = false)` which will cause API doc errors.